testsuite/features/support/env.rb

```diff
@@ -133,6 +133,14 @@ def enable_assertions
   end
 end
 
+BeforeAll do
+  begin
+    $api_test.auth.login('admin', 'admin')
```

Comment:
AFAIK we have special API tests which use different users. Does this still work? I think you need logout first at these tests, right?

Reply:
None of our calls to login to the API were using other user than "admin", as you can see in the deleted lines.
We might be testing with a different user only for WebUI authentication.